### PR TITLE
ENSIME + fixes for tests

### PR DESCRIPTION
**Pull request by junkumar**
Works with new sbt

### CODE REVIEW

File: src/test/scala/shark/SharkServerSuite.scala [hunk 1]
```diff
@@ -32,10 +35,15 @@ class SharkServerSuite extends FunSuite with BeforeAndAfterAll with ShouldMatche
...
-      "javax.jdo.option.ConnectionURL=jdbc:derby:;databaseName=" + METASTORE_PATH + ";create=true",
-      "-hiveconf",
-      "hive.metastore.warehouse.dir=" + WAREHOUSE_PATH)
+      "--verbose",
+      "-p",
+      PORT,
+      "--hiveconf",
+      "hive.root.logger=INFO,console",
+      "--hiveconf",
+      "\"javax.jdo.option.ConnectionURL=jdbc:derby:;databaseName=" + METASTORE_PATH + ";create=true\"",
```
Comment (rxin) [inline src/test/scala/shark/SharkServerSuite.scala:44]: Don't do the two level quote. Those quotes are only required in bash. When the process is launched via process builder, the entire string is realdy a single unit and  ";" doesn't separate the commands.

**Comment by rxin** [comment]
Thanks. This looks good. Just a small change required.

File: src/test/scala/shark/SharkServerSuite.scala (hunk 1 above)
Comment (junkumar) [inline src/test/scala/shark/SharkServerSuite.scala:44]: Why not leave the 2 level quote in the code?
It works correctly when launched via process builder.
It also has the added advantage of dumping something to the console that can be copied & pasted directly into another shell while debugging when you are not going to remember or notice the quotes.

Comment (rxin) [inline src/test/scala/shark/SharkServerSuite.scala:44]: ok the 2nd point convinced me.

**Pull request merged**